platform-includes/debug-symbols/dart-plugin/flutter.mdx

```diff
@@ -0,0 +1,162 @@
+
+## Building Your Application
+
+Before running the plugin, build your Flutter application with one of the following commands. Obuscated is encouraged for production builds, and will make uploading debug symbols necessary to get readable stack traces.
+
+```bash {tabTitle: Obfuscated}
+flutter build apk --obfuscate --split-debug-info=<output-directory>
+flutter build ios --obfuscate --split-debug-info=<output-directory>
+flutter build macos --obfuscate --split-debug-info=<output-directory>
+flutter build windows --obfuscate --split-debug-info=<output-directory>
+flutter build linux --obfuscate --split-debug-info=<output-directory>
+```
+
+```bash {tabTitle: Standard}
+flutter build apk
+flutter build ios
+flutter build macos
+flutter build windows
+flutter build linux
+```
+
+```bash {tabTitle: Flutter Web}
+flutter build web --release --source-maps
+```
+<Alert>
+For Flutter web run `flutter build web --release --source-maps` to generate source maps.
+</Alert>
+
+## Running the Plugin
+
+After building your application, run the plugin to upload debug symbols:
+
+```bash
+flutter pub run sentry_dart_plugin
+```
+
+<Alert>
+
+If you don't obfuscate your build, the plugin won't upload debug symbols but will still configure source context if enabled.
+
+</Alert>
+
+## Android ProGuard Integration
+
+If you have ProGuard (`minifyEnabled`) enabled, you must upload Android Proguard/R8 mapping files. You have two options:
+
+1. **Use the Sentry Android Gradle Plugin (Recommended)**
+   
+   After installing the [Sentry Android Gradle Plugin](/platforms/android/configuration/gradle/), set `autoInstallation` to `false` in your `app/build.gradle` file:
+
+   ```groovy {filename: app/build.gradle}
+   sentry {
+       autoInstallation {
+         enabled = false
+       }
+   }
+   ```
+   
+   This is necessary because Sentry Flutter already ships with a compatible Sentry Android SDK, so we need to avoid conflicts.
+   
+   Follow the [Android Gradle Plugin guide](/platforms/android/configuration/gradle/#proguardr8--dexguard) to complete the ProGuard mapping setup.
+
+2. **Use the Sentry CLI**
+   
+   Alternatively, you can use the [Sentry CLI](/cli/dif/#uploading-files) to manually upload mapping files.
+
+<Alert>
+
+Sentry's Flutter SDK doesn't currently support the `uploadNativeSymbols` flag from the Sentry Gradle Plugin.
+
+</Alert>
+
+## iOS and macOS dSYM Uploads
+
+For iOS and macOS applications, you need to upload dSYM files. The Sentry Dart Plugin will automatically handle this when you build with the `--obfuscate` flag.
+
+For non-obfuscated builds or additional configuration options, you can:
+
+1. **Use the Sentry Cocoa script**
+   
+   Follow the [Sentry Cocoa debug symbols guide](/platforms/apple/guides/ios/dsym/) to set up automatic dSYM uploads with the script.
+
+2. **Use the Sentry CLI**
+   
+   Use the [Sentry CLI](/platforms/apple/guides/ios/dsym/#upload-dsyms-with-cli) to manually upload dSYM files.
```

Comment:
we can actually remove this, this is instruction for manual upload which is in the manual upload page that you created